lib/computer.ex

```diff
@@ -3,17 +3,23 @@ defmodule Computer do
   defp other_mark(:o), do: :x
 
   def move(board, mark) do
-    if board == Board.empty_board do
-      random_move()
+    if first_move(board, 0) do
```

Comment:
Yeah, I'm still looking for a better name which should mean: numbers of random move before to use another strategy.

Some of my tests doesn't expect to receive a random position has they tests minimax, that's why I put 0 by default